types/index.d.ts

```diff
@@ -0,0 +1,26 @@
+import { Client } from '@sentry/types';
+
+// add type to Vue context
+declare module 'vue/types/vue' {
+  interface Vue {
+    $sentry?: Client;
+  }
+}
+
+// App Context and NuxtAppOptions
+declare module '@nuxt/types' {
+  interface Context {
+    readonly $snack?: Client;
```

Comment:
`$snack`?

Reply:
🙃

Reply:
uha ;) sorry about that, fixed it. 
i rebased and squashed all the commits, unfortuantely I guess due to missing write permissions this is not reflected in this pr.

Reply:
checkout #136 for a possible solution to this ;)